modules/ztp-acm-installing-disconnected-rhacm.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="installing-disconnected-rhacm_{context}"]
+= Installing Red Hat Advanced Cluster Management in a disconnected environment
+
+You use {rh-rhacm-first} on a hub cluster in the disconnected environment to manage the deployment of distributed unit (DU) profiles on multiple managed spoke clusters.
+
+.Prerequisites
+
+* Install the {product-title} CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Configure a disconnected mirror registry for use in the cluster.
```

Comment:
Sorry about the last-minute changes but we need to add a note that any operators the customer wants to deploy to their spoke clusters must also be added/mirrored to this registry.

Reply:
@CrystalChun isn't this covered by link in next step to install OLM for disconnected env? https://docs.openshift.com/container-platform/4.6/operators/admin/olm-restricted-networks.html

Reply:
Or are you just molting that customers have custom operators they develop themselves?

Reply:
More of an FYI or a note for the customer in case it's not obvious that they need to add operators for the spoke cluster even if they're not deploying those operators to the Hub. 

I'm not sure I see spoke clusters mentioned in the OLM link. Could you point it out if I missed it?

Reply:
Ok thanks @CrystalChun.

Reply:
@StephenJamesSmith see above for Crystal's suggestion.

Reply:
Added this Note for Aidan under the prereq item on L14:  "If you want to deploy Operators to the spoke clusters you must also add them to this registry."